waspc/src/Wasp/JsImport.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.JsImport
+  ( JsImport (..),
+    JsImportName (..),
+    JsImportData,
+    JsImportAlias,
+    JsImportPath,
+    getJsImportData,
+  )
+where
+
+import Data.Data (Data)
+import Data.List (isPrefixOf)
+import StrongPath (Dir', File', Path, Posix, Rel)
+import qualified StrongPath as SP
+
+data JsImport = JsImport
```

Comment:
Ok, so this is general representation of JS import.

I would put a comment here explaining that, plus explaining that it is simplified and has can import only one name (be it default or listed) + potentially alias.
The reason for that is that we don't need more at the moment.